cmd/manager/main.go

```diff
@@ -373,3 +331,84 @@ func initCustomZapLogger(level, encoding string) (*zap.Logger, error) {
 	}
 	return cfg.Build()
 }
+
+func checkAndEnableDBaaS(mgr manager.Manager, logger *zap.Logger, config Config) (bool, error) {
```

Comment:
can move this function into separate files just looks more clean ? WDYT ?

Reply:
done